VideoLocker/src/main/java/org/edx/mobile/util/images/ErrorUtils.java

```diff
@@ -16,31 +17,43 @@
 
     protected static final Logger logger = new Logger(ErrorUtils.class.getName());
 
+    public static class Error {
+        public MessageType type;
+        public String message;
+    }
```

Comment:
Looking at it again, since, we are not introducing an `enum` for a `Task`'s type, having an `Error` class here just for the purpose of encapsulating a `String` and `MessageType` seems unnecessary.

So, I am getting rid of it altogether and reverting to the `getErrorMessage` function.